utils/doclint/documentation.js

```diff
@@ -314,8 +316,11 @@ Documentation.Member = class {
       this.args.set(arg.name, arg);
       arg.enclosingMethod = this;
       if (arg.name === 'options') {
+        arg.type.properties.forEach((p,i) => {
+          p.enclosingMethod = this,
+          p.order = p.order || i + 1;
```

Comment:
How about keeping properties in the original order and sorting them where needed (just have `sortedProperties()` method on `type`) would that be equivalent? If it doesn't work let's at least rename order to something more specific like `definitionOrder`.

Reply:
I think we don't need a `sortedProperties` on our side. But this would affect the generators on Java and Python, right?

Do you want me to remove this and create issues on both repos to check the impact?

Reply:
Java and Python generate from api.json at the moment so we just need to make sure that the order is preserved in the output of generateApiJson.js

Reply:
will do!